src/bindings/python/build_linux.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -o errexit -o pipefail -o nounset
+
+BUILD_DIR="${1:-build_manylinux}"
+# if someone specifies the python version, we'll build this wheel to ./wheelhouse
+PYTHON_VERSION="${2:-}"
+
+echo "[INFO] ccache dir is $(ccache -k cache_dir) with CCACHE_DIR=${CCACHE_DIR:-}"
```

Comment:
we set `CCACHE_DIR`  in GHA so the `manylinux image caches to `/host/root/.cache/ccache` where `/host` is the mounting point for the whole CI machine